src/options.h

```diff
@@ -30,13 +30,16 @@ class options_manager
                 }
         };
         static std::vector<id_and_option> lang_options;
+        static std::vector<id_and_option> actual_lang_options;
```

Comment:
Can this be `const`?  IT looks like it can be, and that is preferable for global variables like this.